eventuals/task.h

```diff
@@ -448,28 +472,73 @@ struct _TaskFromToWith final {
 template <
     typename From_,
     typename To_,
+    typename Errors_,
     typename... Args_>
-class TaskFromToWith {
+class TaskFromToWith final {
  public:
   template <typename Arg>
   using ValueFrom = To_;
 
-  TaskFromToWith(MonostateIfVoidOrReferenceWrapperOr<To_> value)
-    : e_(std::move(value)) {}
+  template <typename Arg, typename Errors = std::tuple<>>
```

Comment:
No default here, someone should always give us the errors to start with.
```suggestion
  template <typename Arg, typename Errors>
```